### PR TITLE
Signup Flow: Social First signup

### PR DESCRIPTION
**Pull request by escapemanuele**
More details in this issue: https://github.com/Automattic/dotcom-forge/issues/3893#issuecomment-1731924438

## Before

image

## After

image

## Testing
1. Live Link
2. Go through some flows like[ Link In Bio](http://calypso.localhost:3000/setup/link-in-bio) and reach user signup
3. Try Social Logins and email passwordless registration

### CODE REVIEW

File: client/blocks/signup-form/signup-form-social-first.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+					socialServiceResponse={ socialServiceResponse }
+					isReskinned={ isReskinned }
+					redirectToAfterLoginUrl={ redirectToAfterLoginUrl }
+					disableTosText={ true }
+					compact={ true }
+				>
+					<button
+						className="social-buttons__button button"
+						onClick={ () => setCurrentStep( 'email' ) }
+					>
```
Comment (alshakero) [inline client/blocks/signup-form/signup-form-social-first.tsx]: Shouldn't we use a pre-styled Calypso `Button` ?

**Review by alshakero — COMMENTED** [review]
Testing now. I'll re-review tomorrow with fresh eyes, but all I can find now is nits. Great job!!

**Review by alshakero — COMMENTED** [review]
This form should have the same width as the one before it, 327px.

image

**Comment by escapemanuele** [comment]
Thanks for the feedback, I think I addressed it all ✌️ @alshakero

**Comment by agrullon95** [comment]
@escapemanuele - Lets update the position of the "Log in" text on mobile to match the designs:

CleanShot 2023-09-27 at 14 12 38@2x

**Comment by escapemanuele** [comment]
Thank you @agrullon95, I moved the Log In link 👍

**Pull request merged**